tests/async_tests/test_interfaces.py

```diff
@@ -228,6 +228,10 @@ async def test_proxy_socket_does_not_leak_when_the_connection_hasnt_been_added_t
         *filter(lambda warn: "asyncio" not in warn.filename, recorded_warnings.list)
     ]
 
+    if warnings:
```

Comment:
I'm not familiar with the project code, but this code looks meaningless

Reply:
It's for CI debugging. You can check the last failed build, this line prints a warning. It means that I forgot to close a connection in the newly added test

Reply:
the pytest will show variables when assertion fail.